### PR TITLE
Add tiflash cdc for performance overview

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12240
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-performance-overview-dashboard.md [hunk 1]
```diff
@@ -141,14 +158,61 @@ All these three metrics include the average duration and P99 duration in all TiK
...
+- Request QPS: The number of coprocessor requests received by all TiFlash instances.
+
+    - `batch`: Number of batch requests.
+    - `batch_cop`: Number of coprocessor requests in the batch requests.
+    - `cop`: Number of coprocessor requests that are sent directly via the coprocessor interface.
+    - `cop_dag`: Number of dag requests in all coprocessor requests.
+    - `super_batch`: number of requests that enable the Super Batch feature.
+- Executor QPS: The number of each type of dag executors in the requests received by all TiFlash instances. `table_scan` is the table scan executor. `selection` is the selection executor. `aggregation` is the aggregation executor. `top_n` is the `TopN` executor. `limit` is the limit executor.
+- Request Duration Overview: Provides a stacked graph of total processing time per second for all request types in all TiFlash instances.
+- Request Duration: The total processing duration for each MPP type and coprocessor request type in all TiFlash instances. The total processing duration is from the time that the coprocessor request is received to the time that the response of the request is completed, which includes the average and p99 latency.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:188]:
```suggestion
- Request Duration: The total processing duration for each MPP and coprocessor request type in all TiFlash instances. It is from the time that the coprocessor request is received to the time that the response of the request is completed, which includes the average latency and p99 latency.
```

File: grafana-performance-overview-dashboard.md [hunk 2]
```diff
@@ -141,14 +158,61 @@ All these three metrics include the average duration and P99 duration in all TiK
...
+- Request Handle Duration: The actual processing duration for each MPP type and coprocessor request type in all TiFlash instances. The actual processing duration is from starting to execute the coprocessor request to completing the execution, which includes the average and p99 latency.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:189]:
```suggestion
- Request Handle Duration: The actual processing duration for each MPP and coprocessor request type in all TiFlash instances. It is from the start of executing the coprocessor request to the completion of the execution, which includes the average latency and p99 latency.
```

File: grafana-performance-overview-dashboard.md [hunk 3]
```diff
@@ -141,14 +158,61 @@ All these three metrics include the average duration and P99 duration in all TiK
...
+- Raft Wait Index Duration: The time used by `wait_index` for all TiFlash instances, namely the time used to wait until Region index >= `read_index` after the `read_index` request is received.
+- Raft Batch Read Index Duration: The time used by `read_index` for all TiFlash instances. Most time is used for interaction with the Region leader and retry.
+- Write Throughput By Instance: The throughput of write by instance. It includes the throughput by applying the Raft write commands and Raft snapshots.
+- Write flow: The traffic of disk writes by all TiFlash instances.
+- Read flow: The traffic of disk reads by all TiFlash instances.
+
+## CDC
+
+- CPU usage: The CPU usage per TiCDC node.
+- Memory usage: The memory usage per TiFlash node.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:199]:
```suggestion
- Memory usage: The memory usage per TiCDC node.
```

File: grafana-performance-overview-dashboard.md [hunk 4]
```diff
@@ -141,14 +158,61 @@ All these three metrics include the average duration and P99 duration in all TiK
...
+- Changefeed resolved ts lag: The progress lag of data replication (the unit is second) between the upstream and TiCDC nodes.
+- The status of changefeeds:
+
+    - 0: Normal
+    - 1: Error
+    - 2: Failed
+    - 3: Stopped
+    - 4: Finished
+    - -1: Unknown
+- Puller output events/s: The number of events that the Puller module of TiCDC nodes sends to the Sorter module per second.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:211]:
```suggestion
- Puller output events/s: The number of rows that the Puller module of TiCDC nodes sends to the Sorter module per second.
```

File: grafana-performance-overview-dashboard.md [hunk 5]
```diff
@@ -141,14 +158,61 @@ All these three metrics include the average duration and P99 duration in all TiK
...
+- Sorter output events/s: The number of rows that the Sorter module of TiCDC nodes sends to the Mounter module per second.
+- Mounter output events/s: The number of rows that the Mounter module of TiCDC nodes sends to the Sink module per second.
+- Table sink output events/s: The number of rows that the Table Sorter module of the TiCDC nodes send to the Sink module per second.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:214]:
```suggestion
- Table sink output events/s: The number of rows that the Table Sorter module of the TiCDC nodes sends to the Sink module per second.
```

File: grafana-performance-overview-dashboard.md [hunk 6]
```diff
@@ -141,14 +158,61 @@ All these three metrics include the average duration and P99 duration in all TiK
...
+- SinkV2 - Sink flush rows/s: The number of rows that the Sink module in the TiCDC node sends to the downstream per second.
+- Transaction Sink Full Flush Duration: The average latency and p999 latency that is taken by the MySQL Sink of TiCDC nodes to writing downstream transactions.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:216]:
```suggestion
- Transaction Sink Full Flush Duration: The average latency and p999 latency of writing downstream transactions by the MySQL Sink of TiCDC nodes.
```

File: grafana-performance-overview-dashboard.md [hunk 7]
```diff
@@ -141,14 +158,61 @@ All these three metrics include the average duration and P99 duration in all TiK
...
+- MQ Worker Send Message Duration Percentile: The latency of sending messages by MQ worker when the downstream is Kafka.
+- Kafka Outgoing Bytes: The bytes of writing transactions to downstream for MQ Workload.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:218]:
```suggestion
- Kafka Outgoing Bytes: The traffic of writing downstream transactions in MQ Workload.
```

File: performance-tuning-methods.md [hunk 8]
```diff
@@ -125,6 +125,14 @@ The diagrams of database time breakdown and execution time overview present both
 - Database Time by SQL Phase: Most time is consumed in the execute phase in green.
 - SQL Execute Time Overview: The KV request PessimisticLock shown in red consumes the most time during SQL execution, and the execution time is obviously longer than the total time of KV requests. This is caused by serious lock conflicts in write statements and frequent lock retries prolong `Retried execution time`. Currently, TiDB does not measure `Retried execution time`.
 
+**Example 5: HTAP CH-Benchmark Load**
```
Comment (qiancai) [inline performance-tuning-methods.md:128]:
```suggestion
**Example 5: HTAP CH-Benchmark workload**
```

File: performance-tuning-methods.md [hunk 9]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
 
 ![OLTP-Prepared](/media/performance/oltp_prepared_statement_no_plan_cache.png)
 
-#### KV/TSO Request OPS and connection information
+**Example 4: Prepared statements have a resource leak**
+
+The number of `StmtExecute` commands per second is much greater than that of `StmtClose` per second, which indicates that the application has a object leak for prepared statements.
```
Comment (qiancai) [inline performance-tuning-methods.md:186]:
```suggestion
The number of `StmtPrepare` commands per second is much greater than that of `StmtClose` per second, which indicates that the application has an object leak for prepared statements.
```

File: performance-tuning-methods.md [hunk 10]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
+
+![OLTP-Query](/media/performance/prepared_statement_leaking.png)
+
+- In the QPS panel, the red bold line indicates the number of failed queries, and the Y axis on the right indicates the coordinate value of the number. In this example, the number of failed quries per second is 74.6.
+- In the CPS By Type panel, the number of `StmtExecute` commands per second is much greater than that of `StmtClose` per second, which indicates that a object leak occurs in the application for prepared statements.
```
Comment (qiancai) [inline performance-tuning-methods.md:191]:
```suggestion
- In the CPS By Type panel, the number of `StmtPrepare` commands per second is much greater than that of `StmtClose` per second, which indicates that an object leak occurs in the application for prepared statements.
```

File: performance-tuning-methods.md [hunk 11]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
+- In the Queries Using Plan Cache OPS panel, `avg-miss` is almost equal to `StmtExecute` in the CPS By Type panel, which indicates that almost all SQL executions miss the execution plan cache.
```
Comment (qiancai) [inline performance-tuning-methods.md:192]:
```suggestion
- In the Queries Using Plan Cache OPS panel, `avg-miss` is almost equal to `StmtExecute`, which indicates that almost all SQL executions miss the execution plan cache.
```

File: performance-tuning-methods.md [hunk 12]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
 
-In the KV/TSO Request OPS panel, you can view the statistics of KV and TSO requests per second. Among the statistics, `kv request total` represents the sum of all requests from TiDB to TiKV. By observing the types of requests from TiDB to PD and TiKV, you can get an idea of the workload profile within the cluster.
+#### KV/TSO Request OPS and KV Request Time By Source
 
-In the Connection Count panel, you can view the total number of connections and the number of connections per TiDB. The counts help you determine whether the total number of connections is normal and the number of connections per TiDB is even. `active connections` records the number of active connections, which is equal to the database time per second.
+- In the KV/TSO Request OPS panel, you can view the statistics of KV and TSO requests per second. Among the statistics, `kv request total` represents the sum of all requests from TiDB to TiKV. By observing the types of requests from TiDB to PD and TiKV, you can get an idea of the workload profile within the cluster.
+- In the KV Request Time By Source panel, you can view the time ratio of each KV request and all request sources.
```
Comment (qiancai) [inline performance-tuning-methods.md:197]:
```suggestion
- In the KV Request Time By Source panel, you can view the time ratio of each KV request type and all request sources.
```

File: performance-tuning-methods.md [hunk 13]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
+    - kv request total time: The total time of processing KV and TiFlash requests per second.
+    - Each KV request and the corresponding request source form a columnar stacked graph, in which `external` identifies normal business requests and `internal` identifies internal activity requests (such as DDL and auto analyze requests).
+
+In the KV Request Time By Source panel, you can view the total number of connections and the number of connections per TiDB. 
+
+The counts help you determine whether the total number of connections is normal and the number of connections per TiDB is even. `active connections` records the number of active connections, which is equal to the database time per second.
```
Comment (qiancai) [inline performance-tuning-methods.md:203]:
```suggestion
```

File: performance-tuning-methods.md [hunk 14]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
+- The total number of KV requests per second is 7,970. The top request types are `Prewrite`, `Commit`, `PessimisticsLock`, and `BatchGet` in order of number of requests.
+- Most of the KV processing time is spent on `Commit-external_Commit` and `Prewrite-external_Commit`, which indicates that the most time-consuming KV requests are `Commit` and `Prewrite` from external commit statements.
 
-**Example 2: Idle workload**
+**Example 2: Analyze workload**
 
-![OLTP](/media/performance/cloud_long_idle_kv_conn.png)
+![OLTP](/media/performance/internal_stats.png)
 
 In this workload:
```
Comment (qiancai) [inline performance-tuning-methods.md:218]:
```suggestion
In this workload, only `ANALYZE` statements are running in the cluster:
```

File: performance-tuning-methods.md [hunk 15]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
 
-- The total number of KV requests per second is 2600 and the number of TSO requests per second is 1100.
-- The total number of connections is 410, which are evenly distributed in three TiDB instances. The number of active connections is only 2.5, indicating that the database system is relatively idle.
+- The total number of KV requests per second is 3050 and the number of Cop requests per second is 1100.
```
Comment (qiancai) [inline performance-tuning-methods.md:220]:
```suggestion
- The total number of KV requests per second is 35.5 and the number of Cop requests per second is 9.3.
```

File: performance-tuning-methods.md [hunk 16]
```diff
@@ -168,29 +181,44 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
+- Most of the KV processing time is spent on `Cop-internal_stats`, which indicates that the most time-consuming KV request is `Cop` from internal analyze operations.
```
Comment (qiancai) [inline performance-tuning-methods.md:221]:
```suggestion
- Most of the KV processing time is spent on `Cop-internal_stats`, which indicates that the most time-consuming KV request is `Cop` from internal `ANALYZE` operations.
```

File: performance-tuning-methods.md [hunk 17]
```diff
@@ -245,7 +273,19 @@ In real customer scenarios, it is not rare that the bottleneck is outside the da
 - The network latency from the application server to the database is high. For example, the network latency is high because in public-cloud deployments the applications and the TiDB cluster are not in the same region, or the dns workload balancer and the TiDB cluster are not in the same region.
 - The bottleneck is in client applications. The application server's CPU cores and Numa resources cannot be fully utilized. For example, only one JVM is used to establish thousands of JDBC connections to TiDB.
 
-**Example 1: TiDB is the bottleneck of user response time**
+In the Connection Count panel, you can check the total number of connections and also the number of connections on each TiDB node, which helps you evaluate whether the total number of connections is normal and whether the number of connections on each TiDB node is unbalanced. `active connections` indicates the number of active connections, which is equal to the database time per second. The Y axis on the right indicates `disconnection/s`, which indicates the number of disconnections per second in a cluster, which can be used to determine whether the application uses short connections.
```
Comment (qiancai) [inline performance-tuning-methods.md:276]:
```suggestion
In the Connection Count panel, you can check the total number of connections and also the number of connections on each TiDB node, which helps you determine whether the total number of connections is normal and whether the number of connections on each TiDB node is unbalanced. `active connections` indicates the number of active connections, which is equal to the database time per second. The Y axis on the right is for `disconnection/s`, which indicates the number of disconnections per second in a cluster, which can be used to determine whether the application uses short connections.
```

File: performance-tuning-methods.md [hunk 18]
```diff
@@ -245,7 +271,19 @@ In real customer scenarios, it is not rare that the bottleneck is outside the da
 - The network latency from the application server to the database is high. For example, the network latency is high because in public-cloud deployments the applications and the TiDB cluster are not in the same region, or the dns workload balancer and the TiDB cluster are not in the same region.
 - The bottleneck is in client applications. The application server's CPU cores and Numa resources cannot be fully utilized. For example, only one JVM is used to establish thousands of JDBC connections to TiDB.
 
-**Example 1: TiDB is the bottleneck of user response time**
+In the Connection Count panel, you can check the total number of connections and also the number of connections on each TiDB node, which helps you determine whether the total number of connections is normal and whether the number of connections on each TiDB node is unbalanced. `active connections` indicates the number of active connections, which is equal to the database time per second. The Y axis on the right is for `disconnection/s`, which indicates the number of disconnections per second in a cluster, which can be used to determine whether the application uses short connections.
```
Comment (qiancai) [inline performance-tuning-methods.md:274]:
```suggestion
In the Connection Count panel, you can check the total number of connections and also the number of connections on each TiDB node, which helps you determine whether the total number of connections is normal and whether the number of connections on each TiDB node is unbalanced. `active connections` indicates the number of active connections, which is equal to the database time per second. The Y axis on the right indicates the number of `disconnection/s` per second in a cluster, which can be used to determine whether the application uses short connections.
```

File: performance-tuning-methods.md [hunk 19]
```diff
@@ -133,15 +141,20 @@ By checking the following three panels in Performance Overview, you can learn th
...
+
+    - No prepared plan cache is hit: `avg-hit` (the number of hits per second) is 0, and `avg-miss` equal to the number of `StmtExecute` commands per second. The possible reasons include:
+        - The application is using the query interface.
+        - The cached plans are cleaned up by calling the `StmtClose` command after each `StmtExecute` execution.
+        - All statements executed by `StmtExecute` do not meet the [cache conditions](sql-prepared-plan-cache.md) so the execution plan cache cannot be hit.
+    - All prepared plan cache is hit: `avg-hit` (the number of hits per second) is equal to the number of `StmtExecute` commands per second, and `avg-miss` (the number without hits per second) is 0.
+    - Some prepared plan cache is hit: `avg-hit` (the number of hits per second) is fewer than the number of `StmtExecute` commands per second. Prepared plan cache has known limitations: for example, it does not support subqueries, so SQL statements with subqueries cannot use prepared plan cache.
```
Comment (ran-huang) [inline performance-tuning-methods.md:153]:
```suggestion
    - Some prepared plan cache is hit: `avg-hit` (the number of hits per second) is fewer than the number of `StmtExecute` commands per second. Prepared plan cache has known limitations. For example, it does not support subqueries, so SQL statements with subqueries cannot use prepared plan cache.
```

File: performance-tuning-methods.md [hunk 20]
```diff
@@ -168,29 +181,42 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
+#### KV/TSO Request OPS and KV Request Time By Source
 
-In the Connection Count panel, you can view the total number of connections and the number of connections per TiDB. The counts help you determine whether the total number of connections is normal and the number of connections per TiDB is even. `active connections` records the number of active connections, which is equal to the database time per second.
+- In the KV/TSO Request OPS panel, you can view the statistics of KV and TSO requests per second. Among the statistics, `kv request total` represents the sum of all requests from TiDB to TiKV. By observing the types of requests from TiDB to PD and TiKV, you can get an idea of the workload profile within the cluster.
+- In the KV Request Time By Source panel, you can view the time ratio of each KV request type and all request sources.
+    - kv request total time: The total time of processing KV and TiFlash requests per second.
+    - Each KV request and the corresponding request source form a columnar stacked graph, in which `external` identifies normal business requests and `internal` identifies internal activity requests (such as DDL and auto analyze requests).
```
Comment (ran-huang) [inline performance-tuning-methods.md:199]:
```suggestion
    - Each KV request and the corresponding request source form a stacked bar chart, in which `external` identifies normal business requests and `internal` identifies internal activity requests (such as DDL and auto analyze requests).
```

File: performance-tuning-methods.md [hunk 21]
```diff
@@ -168,29 +181,42 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
 **Example 1: Busy workload**
 
-![TPC-C](/media/performance/tpcc_kv_conn.png)
+![TPC-C](/media/performance/tpcc_source_sql.png)
 
 In this TPC-C workload:
 
-- The total number of KV requests per second is 104,200. The top request types are `PessimisticsLock`, `Prewrite`, `Commit` and `BatchGet` in order of number of requests.
-- The total number of connections is 810, which are evenly distributed in three TiDB instances. The number of active connections is 787.1. Therefore, 97% of the connections are active, indicating that the database is the bottleneck for this system.
+- The total number of KV requests per second is 7,970. The top request types are `Prewrite`, `Commit`, `PessimisticsLock`, and `BatchGet` in order of number of requests.
```
Comment (ran-huang) [inline performance-tuning-methods.md:209]: The Chinese version is "79.7K".
```suggestion
- The total number of KV requests per second is 79,700. The top request types are `Prewrite`, `Commit`, `PessimisticsLock`, and `BatchGet` in order of number of requests.
```

File: performance-tuning-methods.md [hunk 22]
```diff
@@ -245,7 +271,19 @@ In real customer scenarios, it is not rare that the bottleneck is outside the da
 - The network latency from the application server to the database is high. For example, the network latency is high because in public-cloud deployments the applications and the TiDB cluster are not in the same region, or the dns workload balancer and the TiDB cluster are not in the same region.
 - The bottleneck is in client applications. The application server's CPU cores and Numa resources cannot be fully utilized. For example, only one JVM is used to establish thousands of JDBC connections to TiDB.
 
-**Example 1: TiDB is the bottleneck of user response time**
+In the Connection Count panel, you can check the total number of connections and also the number of connections on each TiDB node, which helps you determine whether the total number of connections is normal and whether the number of connections on each TiDB node is unbalanced. `active connections` indicates the number of active connections, which is equal to the database time per second. The Y axis on the right indicates the number of `disconnection/s` per second in a cluster, which can be used to determine whether the application uses short connections.
```
Comment (ran-huang) [inline performance-tuning-methods.md:274]:
```suggestion
In the Connection Count panel, you can check the total number of connections and also the number of connections on each TiDB node, which helps you determine whether the total number of connections is normal and whether the number of connections on each TiDB node is unbalanced. `active connections` indicates the number of active connections, which is equal to the database time per second. The Y axis on the right (`disconnection/s`) indicates the number of disconnections per second in a cluster, which can be used to determine whether the application uses short connections.
```

File: performance-tuning-methods.md [hunk 23]
```diff
@@ -245,7 +271,19 @@ In real customer scenarios, it is not rare that the bottleneck is outside the da
...
+
+**Example 1: The number of disconnection/s is too high**
+
+![high disconnection/s](/media/performance/high_disconnections.png)
+
+In this workload:
+
+- The average latency and P99 latency of all SQL statements are 10.8 ms and 84.1 ms, respectively.
+- The average connection idle time in transactions `avg-in-txn` is 9.4 ms.
+- The total number of connections to the cluster is 3700, and the number of connections to each TiDB node is 1800. The average number of active connections is 40.3, which indicates that most of connections are idle. The average number of `disonnnection/s` is 55.8, which indicates that the application is connecting and disconnecting frequently. The behavior of short connections will have a certain impact on TiDB resources and response time.
```
Comment (ran-huang) [inline performance-tuning-methods.md:284]:
```suggestion
- The total number of connections to the cluster is 3,700, and the number of connections to each TiDB node is 1,800. The average number of active connections is 40.3, which indicates that most of the connections are idle. The average number of `disonnnection/s` is 55.8, which indicates that the application is connecting and disconnecting frequently. The behavior of short connections will have a certain impact on TiDB resources and response time.
```

File: performance-tuning-methods.md [hunk 24]
```diff
@@ -256,9 +294,9 @@ In this TPC-C workload:
 
 The average query latency is significantly greater than `avg-in-txn`, which means the main bottleneck in transactions is inside the database.
 
-**Example 2: The bottleneck of user response time is not in TiDB**
+**Example 3: The bottleneck of user response time is not in TiDB**
```
Comment (ran-huang) [inline performance-tuning-methods.md:297]: To keep consistent with example 2
```suggestion
**Example 3: TiDB is not the bottleneck of user response time**
```

File: performance-tuning-methods.md [hunk 25]
```diff
@@ -133,15 +141,20 @@ By checking the following three panels in Performance Overview, you can learn th
...
 
-    - No prepared plan cache is hit: The number of plan cache hit per second is 0. The application is using the query interface, or cached plans are cleaned up by calling the StmtClose command after each StmtExecute execution.
-    - All prepared plan cache is hit: The number of hits per second is equal to the number of StmtExecute commands per second.
-    - Some prepared plan cache is hit: The number of hits per second is fewer than the number of StmtExecute commands per second. Prepared plan cache has known limitations, for example, it does not support subqueries, SQL statements with subqueries cannot utilize prepared plan cache.
+    `avg-hit + avg-miss` is equal to `StmtExecute`, which is the number of all queries executed per second. When prepared plan cache is enabled in TiDB, the following three scenarios will occur:
+
+    - No prepared plan cache is hit: `avg-hit` (the number of hits per second) is 0, and `avg-miss` equal to the number of `StmtExecute` commands per second. The possible reasons include:
```
Comment (qiancai) [inline performance-tuning-methods.md:148]:
```suggestion
    - No prepared plan cache is hit: `avg-hit` (the number of hits per second) is 0, and `avg-miss` is equal to the number of `StmtExecute` commands per second. The possible reasons include:
```

File: grafana-performance-overview-dashboard.md [hunk 26]
```diff
@@ -67,19 +77,18 @@ Number of queries using plan cache per second in all TiDB instances
...
 Generally, dividing `tso - cmd` by `tso - request` yields the average batch size of requests per second.
 
-## Connection Count
+### KV Request Time By Source
 
-- total: Number of connections to all TiDB instances
-- active connections: Number of active connections to all TiDB instances
-- Number of connections to each TiDB instance
+- kv request total time: Total time of processing KV and TiFlash requests per second in all TiDB instances.
+- Each KV request and the corresponding request source form a stacked bar chart, in which `external` identifies normal business requests and `internal` identifies internal activity requests (such as DDL and auto analyze requests).
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:83]:
```suggestion
- kv request total time: Total time of processing KV and TiFlash requests per second in all TiDB instances
- Each KV request and the corresponding request source form a stacked bar chart, in which `external` identifies normal business requests and `internal` identifies internal activity requests (such as DDL and auto analyze requests)
```

File: performance-tuning-methods.md [hunk 27]
```diff
@@ -168,29 +181,42 @@ In this workload, `Commit QPS` = `Rollback QPS` = `Select QPS`. The application
...
+- In the KV/TSO Request OPS panel, you can view the statistics of KV and TSO requests per second. Among the statistics, `kv request total` represents the sum of all requests from TiDB to TiKV. By observing the types of requests from TiDB to PD and TiKV, you can get an idea of the workload profile within the cluster.
+- In the KV Request Time By Source panel, you can view the time ratio of each KV request type and all request sources.
+    - kv request total time: The total time of processing KV and TiFlash requests per second.
+    - Each KV request and the corresponding request source form a stacked bar chart, in which `external` identifies normal business requests and `internal` identifies internal activity requests (such as DDL and auto analyze requests).
+
+In the KV Request Time By Source panel, you can view the total number of connections and the number of connections per TiDB.
```
Comment (qiancai) [inline performance-tuning-methods.md:201]:
```suggestion
```

File: performance-tuning-methods.md [hunk 28]
```diff
@@ -133,15 +141,20 @@ By checking the following three panels in Performance Overview, you can learn th
...
 
-    - No prepared plan cache is hit: The number of plan cache hit per second is 0. The application is using the query interface, or cached plans are cleaned up by calling the StmtClose command after each StmtExecute execution.
-    - All prepared plan cache is hit: The number of hits per second is equal to the number of StmtExecute commands per second.
-    - Some prepared plan cache is hit: The number of hits per second is fewer than the number of StmtExecute commands per second. Prepared plan cache has known limitations, for example, it does not support subqueries, SQL statements with subqueries cannot utilize prepared plan cache.
+    - No prepared plan cache is hit: `avg-hit` (the number of hits per second) is 0, and `avg-miss` is equal to the number of `StmtExecute` commands per second. The possible reasons include:
+        - The application is using the query interface.
+        - The cached plans are cleaned up because the application calls the `StmtClose` command after each `StmtExecute` execution.
+        - All statements executed by `StmtExecute` do not meet the [cache conditions](sql-prepared-plan-cache.md) so the execution plan cache cannot be hit.
```
Comment (qiancai) [inline performance-tuning-methods.md:151]:
```suggestion
        - All statements executed by `StmtExecute` do not meet the [cache conditions](/sql-prepared-plan-cache.md) so the execution plan cache cannot be hit.
```

File: grafana-performance-overview-dashboard.md [hunk 29]
```diff
@@ -99,39 +104,46 @@ Generally, dividing `tso - cmd` by `tso - request` yields the average batch size
...
 
 - avg-in-txn: Average connection idle duration when the connection is within a transaction
 - avg-not-in-txn: Average connection idle duration when the connection is not within a transaction
 - 99-in-txn: P99 connection idle duration when the connection is within a transaction
+
+### Connection Count
+
+- total: Number of connections to all TiDB instances
+- active connections: Number of active connections to all TiDB instances
+- tidb-\<node-number\>-peer: Number of connections to each TiDB instance
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:119]:
```suggestion
- tidb-{node-number}-peer: Number of connections to each TiDB instance
```

**Review by ran-huang — APPROVED** [review]

**Review by Yui-Song — APPROVED** [review]

**Pull request merged**